controllers/delete_pods_for_buggification.go

```diff
@@ -76,7 +68,10 @@ func (d deletePodsForBuggification) reconcile(ctx context.Context, r *Foundation
 			}
 		}
 
-		shouldCrashLoop := crashLoopAll || crashLoopPods[processGroup.ProcessGroupID]
+		shouldCrashLoop := crashLoopAll
+		if !shouldCrashLoop {
+			_, shouldCrashLoop = crashLoopPods[processGroup.ProcessGroupID]
+		}
```

Comment:
What benefit are we getting by doing this ?
If `crashLoopAll` is true we will not check `crashLoopPods[processGroup.ProcessGroupID]` else we will. The current implementation added one extra check with `if` condition.